lib/CampMinder/SignedRequestFactory.rb

```diff
@@ -21,11 +21,11 @@ def get_payload(signed_payload)
   end
 
   def encode_signature(encoded_payload)
-    prepare_encoded_base64(Base64.encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new('sha256'), @secret_code, encoded_payload))).strip()
+    prepare_encoded_base64(Base64.strict_encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new('sha256'), @secret_code, encoded_payload))).strip()
```

Comment:
Line is too long. [142/80]<br>Do not use parentheses for method calls with no arguments.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.